build.gradle

```diff
@@ -278,7 +278,7 @@ dependencies {
     }
     compile 'org.jooq:jool-java-8:0.9.14'
 
-    testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '3.6.0'
+    testCompile group: 'uk.gov.hmcts', name: 'befta-fw', version: '3.6.0-master-def-fix'
```

Comment:
3.6.1 is now ready. please use that one.

Reply:
👍